tchannel/tchannel.py

```diff
@@ -268,7 +268,7 @@ def advertise(self, routers, name=None, timeout=None):
             other services will use to make contact with you.
 
         :param timeout:
-            The timeout (in seconds) for the initial advertise attempt.
+            The timeout (in sec) for the initial advertise attempt.
```

Comment:
this timeout is not for TTL. it is the advertise timeout. advertise timeout is the advertise behavior not the request.